docs/scenarios_authoring/RRDataSourcesAndSinks.md

```diff
@@ -2,42 +2,73 @@
 sidebar_position: 4
 ---
 
-# Request-Response sources and sinks
+# Request-Response specific components
 
-## Http request as input and output to Nussknacker scenarios
+## Concepts
 
-### Concepts
-
-In Request-Response mode you interact with Nussknacker Lite runtime engine in a similar way as with any "server" in the
+### General
+In Request-Response mode you interact with Nussknacker Lite runtime engine in the same way as with any "server" in the
 Internet. You send request which contains request parameters and you get response from the server. The protocol for this
-communication is HTTP - the same protocol is used by browsers when they communicate with web servers. One of the
-differences is that the content of the communication with the Nussknacker Lite engine ("the server") is encoded as JSON,
-not HTML. Not very surprisingly, in Request-Response processing mode the only possible Source component is Request and
-the only Sink component is Response.
+communication is HTTP - the same protocol is used by browsers when they communicate with web servers. When the  browser is used to browse internet pages, the web server responds with content encoded as HTML. In Nussknacker case, the Nussknacker Lite engine ("the server") respondes with content encoded as JSON. You may envoke Nussknacker Lite engine directly from the browser and watch the JSON response in the browser window. 
```

Comment:
Currently our requests are POSTs mostly, so I don't think it's so easy without custom plugins...
The idea of web server returning content as HTML is also rather outdated, since more and more pages are invoking REST APIs similar to Nussknacker's :)

Reply:
I simplified this bit and shortend it.